lib/clusterable/holding.rb

```diff
@@ -22,12 +22,11 @@ class Holding
         :condition,
         :gov_doc_flag,
         :mono_multi_serial,
-        :date_received,
         :uuid,
         :issn
       ]
 
-    READER_ATTRS = []
+    READER_ATTRS = [:date_received]
```

Comment:
Maybe worth mentioning that `date_received` has its own writer-method defined later. Here it looks like it's just a reader attribute.

Reply:
I see this as essentially the same as :

```ruby
attr_reader :date_received

def date_received=(date)
   ...
end
```

so I think it's probably fine -- but also fine to add a comment if we find ourselves confused later.

Note `HtItem` also has one of these for collection_code.